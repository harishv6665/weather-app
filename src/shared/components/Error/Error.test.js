import React from "react";
import { shallow } from "enzyme";
import {Error} from "../";

describe("Error tests", () => {

    it("should be defined", () => {
        expect(Error).toBeDefined();
    });

    it("should match to snapshot", () => {
        const wrapper = shallow(<Error />);
        expect(wrapper).toMatchSnapshot();
    });

    it("should match the DOM structure", () => {
        const wrapper = shallow(<Error />);
        expect(wrapper.find("div").length).toBe(1);
        expect(wrapper.find("span").length).toBe(2);
        expect(wrapper.find("div span").length).toBe(2);
        expect(wrapper.find("div span").first().text()).toEqual("");
        expect(wrapper.find("span").at(1).text()).toEqual("OOPS..! Something went wrong, try refreshing the page.");
    });

});
