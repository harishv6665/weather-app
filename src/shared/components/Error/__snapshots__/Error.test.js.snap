// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error tests should match to snapshot 1`] = `ShallowWrapper {}`;
